src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -3905,3 +3905,51 @@ body.tribe-welcome #fs_connect {
 		width: 300px;
 	}
 }
+
+/* Upsell Styling */
```

Comment:
It'll be also a good idea to include these classes in a new `.pcss` to start organizing things here.